_pages/en_US/installing-boot9strap-(3dsropxploitinjector-usm).txt

```diff
@@ -0,0 +1,70 @@
+---
+title: "Installing boot9strap (3DS ROP xPloit Injector-USM)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+
+Now that the hacked payload has been sent, we inject the exploited WiFi profile into your connections list.
+
+Once the WiFi profile has been injected, we will use SAFE_MODE, which is a recovery feature present on all 3DS consoles, to activate the exploited WiFi profile.
+
+For a more technical explanation, see the following links for information on the kartdlphax and unSAFE_MODE exploits: [kartdlphax](https://github.com/PabloMK7/kartdlphax), [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/).
+
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+### What You Need
+
+* The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/download/v0.0.7/SafeB9SInstaller-20170605-122940.zip)
+* The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/download/1.4/boot9strap-1.4.zip)
+* The latest release of [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/v1.3)
+* The latest release of [Luma3DS](https://github.com/Luma3DS/releases/latest)
+
+### Instructions
+
+#### Section I - Injecting unSAFE_MODE
+
+1. Press (X) to inject unSAFE_MODE
+1. If the injection was successful, the screen will turn green and the **target 3DS** will automatically power off
+  + If the screen turns red, power off the target 3DS and start again from the beginning of [Section III - kartdlphax on the previous page](3dsropxploitinjector-(kartdlphax)#section-iv---kartdlphax). If this doesn't work, ask for help at [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp))
+
+You will **not** need to use your **source 3DS** to complete any further steps on this guide. Any further steps should only be completed on the **target 3DS**.
```

Comment:
As this page is also linked to from kartminer7, this line could potentially be really confusing (as this is not applicable to kartminer7)